docs/example-scenario/hpc/video-rendering.md

```diff
@@ -0,0 +1,187 @@
+author: Adam Boeglin, Mike Warrington, and Ben Hummerstone
+ms.date: <publish or update date>
+---
+# 3D video rendering on Azure
+
+3D rendering is a time consuming process that requires a significant amount of CPU time co complete.  On a single machine, the process of generating a video file from static assets can take hours or even days depending on the length and complexity of the video you are producing.  Many companies will purchase either expensive high end desktop computers to perform these tasks, or invest in large render farms that they can submit jobs to.  However, by taking advantage of Azure Batch, that power is available to you when you need it and shuts itself down when you don't, all without any capital investment.
+
+Batch gives you a consistent management experience and job scheduling, whether you select Windows Server or Linux compute nodes. With Batch, you can use your existing Windows or Linux applications, including AutoDesk Maya and Blender, to run large-scale render jobs in Azure.
+
+## Potential industry use cases
+
+Other potential use cases that would have a similar environment include:
+
+* 3D Modeling
+* Visual FX (VFX) Rendering
+* Video transcoding
+* Image processing, color correction, & resizing
+
+## Architecture diagram
+
+The solution diagram below is an example of this solution:
+
+![Architecture overview of the components involved in a Cloud Native HPC solution using Azure Batch][architecture]
+
+## Architecture
+
+This solution covers the workflow when using Azure Batch, the data flows through the solution as follows:
+
+1. Upload input files and the applications to process those files to your Azure Storage account
+2. Create a Batch pool of compute nodes in your Batch account, a job to run the workload on the pool, and tasks in the job.
+3. Download input files and the applications to Batch
+4. Monitor task execution
+5. Upload task output
+6. Download output files
+
+To simplify this process, you could also use the [Batch Plugins for Maya & 3ds Max][batch-plugins]
+
+## Azure Batch: Creating an Azure Batch account and pools manually
+This sample solution is will provide help in learning how Azure Batch works while showcasing Azure Batch Labs as an example SaaS solution that can be developed for your own customers:
+
+[Azure Batch Masterclass][batch-labs-masterclass]
+
+## Azure Batch: Deploying a sample solution using an ARM (Azure Resource Manager) template
+
+The following deployment will deploy:
+  - a new Azure Batch account
+  - a storage account
+  - a node pool associated with the batch account
+  - the node pool will be configured to use A2 v2 VMs with Canonical Ubuntu images
+  - the node pool will contain 0 VMs initially and will require scaling manually to add VMs
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmikewarr%2Farchitecture-center%2Fdocs%2Fsample-solutions%2Fhpc%2Fbatchcreatewithpools.json" target="_blank">
```

Comment:
we need to move this template to the solutions-architecture repo, and update the link.